content/docs/command-reference/repro.md

```diff
@@ -53,7 +56,8 @@ corresponding commands. <abbr>Outputs</abbr> are deleted from the
 
 It saves all the data files, intermediate or final results into the <abbr>DVC
 cache</abbr> (unless the `--no-commit` option is used), and updates the hash
-values of changed dependencies and outputs in the corresponding stage files.
+values of changed dependencies and outputs in the corresponding stages (or
+`.dvc` files).
```

Comment:
do we need to complicate everything with `.dvc` files?

Reply:
Unfortunately this is a pretty common thing as I just explained. And we no longer call .dvc files "stages" so we need to keep mentioning both some times...

Reply:
In fact I removed the parentheses.

Reply:
let's just use  `... in the DVC files (.dvc, dvc.lock, etc)` ... essence first, details second

Reply:
OK. Updated to "in the DVC files (`dvc.lock` and `.dvc`)" (those are the only 2 that apply here) also in ebc15605.

Reply:
> DVC files (dvc.lock and .dvc)

Wouldn't it be better to just have "dvc.lock and .dvc files" thouogh? Shorter

Reply:
yep, can be that as well

Reply:
Changed. I like it also because so far I've managed to avoid the term "DVC file" It's confusing vs. .dvc file, which is just one type of DVC file...

Reply:
I would even rename the DVC Files and Dirs guide to Internal Files and Dirs... But anyway, moving on!